modules/administration-guide/pages/repairing-the-devworkspace-operator.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator
+:description: Repairing the {devworkspace} Operator
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator_{context}"]
+= Repairing the {devworkspace} Operator
+
```

Comment:
Missing a leading sentence explaining when a user should consider running this procedure: what are the symptoms of a failed {devworkspace} operator leading to the necessity of a repair?

Reply:
And maybe: what are the most common causes leading to a broken operator?

Reply:
Basically, the symptoms are 2 csv on the cluster in the 'Replacing' and 'Pending' states.

Reply:
I added a leading paragraph:
```
Under certain conditions, such as https://docs.openshift.com/container-platform/4.10/operators/understanding/olm/olm-understanding-olm.html[OLM] restart or cluster upgrade, the {prod2} Operator for {prod-short} might automatically install the {devworkspace} Operator even when it is already present on the cluster. In that case, you can repair the {devworkspace} Operator on OpenShift as follows:
```